src/server/dfly_main.cc

```diff
@@ -724,7 +724,27 @@ void ParseFlagsFromEnv() {
   }
 }
 
+// Sets global variables for executable and config file path for info reporting.
+void ParseFlagfileAndSetGlobals(int argc, char* argv[]) {
```

Comment:
You do not need all this.
You can fetch a value of any flag via reflection. See config_registry for how we read a flag.

```cc
absl::CommandLineFlag* flag2 = absl::FindCommandLineFlag("flagfile");
LOG(ERROR) << "value: " << flag2->CurrentValue();
```